proxy/logstats.cc

```diff
@@ -1164,16 +1167,64 @@ update_schemes(OriginStats *stat, int scheme, int size)
 }
 
 ///////////////////////////////////////////////////////////////////////////////
+// Finds or creates a stats structures if missing
+OriginStats *
+find_or_create_stats(const char *key)
+{
+  OriginStats *o_stats = NULL;
+  OriginStorage::iterator o_iter;
+  char *o_server = NULL;
+
+  // TODO: If we save state (struct) for a run, we probably need to always
+  // update the origin data, no matter what the origin_set is.
```

Comment:
It was an existing comment which seemed reasonable to keep after my minor code refactoring (same functionality was needed for the new option). 
Filed [TS-5072](https://issues.apache.org/jira/browse/TS-5072)